README.md

```diff
@@ -12,6 +12,20 @@ The gem also is dependent on Orangelight to service requests for non-alma IDs. B
 
 You need to have a working version of Chromedriver http://chromedriver.chromium.org/ on your path to successfully run the feature tests associated with this project. Homebrew is a good way to accomplish this. It is currently part of https://github.com/Homebrew/homebrew-cask. 
 
+## External Interfaces
+
+```mermaid
+  graph TD;
+      A[Requests]-->B[Alama EDD and Print];
+      A-->C[Recap/SCSB EDD and Print];
```

Comment:
cool! first time I'm seeing this :) 

Alama - change to -> Alma

Reply:
@christinach I had never seen it, until @jrgriffiniii put it into slack.  I fixed up the spelling issue